docker-compose/oidc/README.adoc

```diff
@@ -0,0 +1,445 @@
+| Tenant ID | App Registration Overview | `9a95fd9e-005d-487a-9a01-d08c1eab2757`
+| Client Secret | Certificates & Secrets | `GENERATED_SECRET_VALUE`
+| Issuer URL | Manual: use Tenant ID | `https://login.microsoftonline.com/<tenant-id>/v2.0`
+|===
+
+[[verify-token-configuration]]
+=== Verify the token configuration
+
+This step confirms you receive a token with expected claims.
+
+. Run a test OAuth client credentials flow:
++
+[,bash]
+----
+curl -X POST "https://login.microsoftonline.com/<tenant-id>/oauth2/v2.0/token" \
+  -d "client_id=<client-id>" \
+  -d "scope=api://<client-id>/.default" \
+  -d "client_secret=<client-secret>" \
+  -d "grant_type=client_credentials"
+----
+
+. Decode the returned token at https://jwt.io[jwt.io^] and verify:
++
+* `iss`: matches the issuer URL
+* `aud`: matches your client ID
+* `ver`: is `2.0`
+* `sub`: is present
+
+. Save the `sub` claim value for later use.
+
+=== App registration final checklist
+
+* [x] `accessTokenAcceptedVersion` set to `2`
+* [x] Custom scope `entraid.v2-access-tokens` created
+
+== Run the lab
+
+. Clone this repository:
++
+```bash
+git clone https://github.com/redpanda-data/redpanda-labs.git
+```
+
+. Change into the `docker-compose/oidc/` directory:
++
+[,bash]
+----
+cd redpanda-labs/docker-compose/oidc
+----
+
+. Set the `REDPANDA_VERSION` environment variable to a version of Redpanda. For all available versions, see the https://github.com/redpanda-data/redpanda/releases[GitHub releases].
++
+For example:
++
+[,bash,subs="attributes+"]
+----
+export REDPANDA_VERSION=v{latest-redpanda-version}
+----
+
+. Set the `REDPANDA_CONSOLE_VERSION` environment variable to the version of Redpanda Console that you want to run. For all available versions, see the https://github.com/redpanda-data/redpanda/releases[GitHub releases].
++
+NOTE: You must use at least version v3.1.0 of Redpanda Console to deploy this lab.
++
+For example:
++
+[,bash,subs="attributes+"]
+----
+export REDPANDA_CONSOLE_VERSION=v{latest-console-version}
+----
+
+. Create a `.env` file and add the following environment variables:
++
+.`.env`
+[,bash,role="no-wrap"]
+----
+TENANT_ID=<tenant-id>
+OIDC_CLIENT_ID=<client-id>
+OIDC_CLIENT_SECRET=<client-secret>
+JWT_SIGNING_KEY=<jwt-signing-key>
+----
++
+Redpanda Console expects configuration to be passed as environment variables, where each YAML key is converted to uppercase and underscores are added for each level of nesting. For example, `authentication.oidc.clientSecret` becomes `AUTHENTICATION_OIDC_CLIENTSECRET`.
++
+To simplify this, user-friendly environment variables, such as `OIDC_CLIENT_SECRET`, are set here and then mapped to the required format in the `docker-compose.yml` file.
++
+This pattern helps separate sensitive or dynamic values from implementation-specific keys. See xref:console:config/configure-console.adoc#environment-variables[Configure Redpanda Console] for details.
```

Comment:
Thinking these links will work after you merge? Forgot this is not in public docs repo--ignore previous comments about links not working.